### PR TITLE
Feature/add linux test to build workflow

### PR DESCRIPTION
**Pull request by Kaloszer**
## Overview/Summary

Draft - I don't seem to be able to execute that workflow on my own fork. Not sure why, I'm mostly an Azure DevOps guy :).
Any clue what I might be missing to test run this workflow?
Also I don't seem to be able to add labels as per the Contributing.md

Edit: Figured it out, I needed to trigger it with a PR on my own fork.

Adds #243.
Requires #290 - otherwise pester test is stuck in an endless loop.

This adds a linux action workflow to run pester tests on a ubuntu machine.

Exec ok:
https://github.com/Kaloszer/PSBicep/actions/runs/5198839942
image

- [x] Checked for duplicate [Pull Requests](https://github.com/PSBicep/PSBicep/pulls)
- [x] Associated it with relevant [issues](https://github.com/PSBicep/PSBicep/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/PSBicep/PSBicep/tree/main)
- [x] Performed testing.
- [ ] Verified build scripts work.
- [ ] Updated relevant and associated documentation.

### CODE REVIEW

File: .github/workflows/RunTests.yaml [hunk 1]
```diff
@@ -27,6 +27,36 @@ jobs:
     - name: Run tests
       run: .\Tests\TestRunner.ps1 -TestResults -Verbosity Normal
       shell: pwsh
+    - name: Publish test results
+      uses: EnricoMi/publish-unit-test-result-action/composite@v1
+      continue-on-error: true
+      with:
+        files: testResults.xml
+
+  build-linux:
```
Comment (StefanIvemo) [inline .github/workflows/RunTests.yaml:36]: (Couldn't leave a comment on a row you didn't change so I'm adding it here) - Can you update the name for the tests on windows from `build` to `build-windows`?

**Comment by Kaloszer** [comment]
Will 'roll-back' the changes tommorow and update the PR.

**Comment by SimonWahlin** [comment]
No worries, I just merged with main.
Looks good! Thanks for your contribution @Kaloszer !

**Review by SimonWahlin — APPROVED** [review]
🚀

**Pull request merged**